### ISSUE TITLE
Identical filename for 01____add-access-rule* in Security and Application rulebase

### ISSUE THREAD

**Issue by pgastinger** [CheckPoint-APIs-Team/ExportImportPolicyPackage#11]
Hi, 

A have an issue with following filenames. If I run your export-script, I get a policyexport.tar.gz, which contains three additional tar.gz-files:

1. export__nat_layer__POLICY__2018_01_18_17_05.tar.gz
2. exported__access_layer__POLICY Security*.tar.gz
3. exported__access_layer__POLICY Application*.tar.gz

I don't have anything to complain about 1.
For 2. and 3., I have two issues:

- a minor one, the naming is not very consistent. In my opinion, you should add` __` after POLICY instead of the space (consistent to 1.)
- if I extract both files (2., 3.), the filenames are identical:
```
-rw-r----- 1 user group    873 Jan 18 17:22 01____add-access-rule__2018_01_18_17_05.json
-rw-r----- 1 user group    507 Jan 18 17:22 01____add-access-rule__2018_01_18_17_05.csv

```
So basically, if I want to export both at the same time, the last extracted files will overwrite the first extracted ones.

Can you please slightly rename those files in the tarballs? Thanks!

best regards, 
Peter

**Comment by CheckPoint-APIs-Team** [comment]
Hi Peter,
Thank you for your comment.
The import/export process doesn't extract the files at the same time, rather one by one - when it finishes with the first, it **deletes** the extracted files and continues to the next - so the files are never overwritten.
Therefore, your concern about exporting both tar files at the same time is gratuitous.

Are you trying to accomplish part of export/import process manually, without the tool involved?

**Comment by CheckPoint-APIs-Team** [comment]
BTW, the difference in the names of NAT rulebase and Access rulebase archives is due to Access Layers.
These are not relevant for NAT.

**Comment by CheckPoint-APIs-Team** [comment]
Thank you Peter!
Your input, comments and insigths are most wellcome.
This shows us what our customers are expecting and how they use these tools.
Your input helps us to improve the functionality of the tool.
